src/main/kotlin/no/nav/syfo/aktivitetskrav/AktivitetskravService.kt

```diff
@@ -31,6 +31,17 @@ class AktivitetskravService(
         )
     }
 
+    fun createAktivitetskrav(personIdent: PersonIdent, previousAktivitetskravUuid: UUID? = null): Aktivitetskrav {
+        val aktivitetskrav = Aktivitetskrav.create(personIdent)
+        val createdAktivitetskrav =
+            aktivitetskravRepository.createAktivitetskrav(aktivitetskrav, previousAktivitetskravUuid)
+        aktivitetskravVurderingProducer.sendAktivitetskravVurdering(
+            aktivitetskrav = createdAktivitetskrav.toAktivitetskrav(),
```

Comment:
Kunne man bare sendt inn `aktivitetskrav` her? Og så kunne vi vel egentlig bare sendt `previousAktivitetskravUuid` videre i stedet for å måtte legge det til i P-objektet og hente det derfra :thinking:

Reply:
Tenker du også å returnert `aktivitetskrav`? Man har jo egentlig ingen garanti for at `aktivitetskrav` og `createdAktivitetskrav` er like, så tenker egentlig det blir best å returnere et domeneobjekt basert på det faktiske objektet som ble lagret. Hva tenker du? 😊

Reply:
Ja, ryddigst å returnere createdAktivitetskrav :+1: